emails/management/commands/process_emails_from_sqs.py

```diff
@@ -36,23 +36,19 @@
 class Command(BaseCommand):
     help = "Fetch email tasks from SQS and process them."
 
-    DEFAULT_BATCH_SIZE = 10
-    DEFAULT_WAIT_SECONDS = 5
-    DEFAULT_VISIBILITY_SECONDS = 120
-
     def __init__(
         self,
         *args,
         batch_size=None,
         wait_seconds=None,
         visibility_seconds=None,
         healthcheck_file=None,
-        delete_failed_messages=False,
+        delete_failed_messages=None,
```

Comment:
suggestion (blocking): This value seems to be all of `None`-type (here), a boolean (in `test_settings`), or an integer (in the `@override_settings` of `test_command_setup_from_settings`? Are these all the same variable controlling the same logic defined by the same setting? If so, can we consolidate its type & arguments?

Reply:
In this case, `None` means "use the value from settings". If we set these to an integer or boolean, we'd be unable to detect that the parameter was not passed.

I've seen other code do something like this:

```python
class Unset:
    """The argument was unset""""
    pass

def __init__(
    self,
    *args,
    batch_size=Unset(),
    wait_seconds=Unset(),
    ...
):
```

Would you prefer something like that?

Reply:
Re-reading this, I think you were specifically calling out `delete_failed_messages`. I still think `None` is the right value here, and I've changed `test_command_setup_from_settings` from `1` to `True`. We can get more precise once mypy is available (PR #1831)

Reply:
Ah, okay that makes more sense to have either `None` or `bool`. Thanks.